testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -1001,6 +1001,22 @@ QUnit.test("option value", function(assert) {
     assert.equal(instance._input().val(), "txt2");
 });
 
+QUnit.test("case sensitive value option", function(assert) {
```

Comment:
I think a name of the test is unclear and does not reflect changes that you have made.
And maybe will be better to move it to dxDropDownList's tests